Chiazokam/Politico
# #163504426 create the edit party endpoint

#### What does this PR do?
This PR tests and creates the edit party Endpont

#### Description of Task to be completed?
- write tests for every test case
- implement admin check with middleware
- create the get one party endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-edit-party-163504426`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `GET` request to `http://localhost:3000/api/v1/parties/1/name`
- You should receive a successful response

#### Any background context you want to provide?
`
{
	 "name": "Damian Party"
}
`

#### What are the relevant pivotal tracker stories?
[#163504426](https://www.pivotaltracker.com/story/show/163504426)

### Images
![image](https://user-images.githubusercontent.com/26940294/51912043-9700e280-23d3-11e9-9b97-a756b7e61e49.png)
